src/api/libcellml/importsource.h

```diff
@@ -22,23 +22,23 @@ limitations under the License.
 namespace libcellml {
 
 /**
- * @brief The Import class.
+ * @brief The ImportSource class.
  *
- * The import class is for encapsulating an external CellML Model.
+ * The import source class is for encapsulating an external CellML Model.
  */
```

Comment:
We need to add more documentation here, @nickerso and I talked about this today I will try and add our thinking on this tomorrow.  I have forgotten and don't want to write the wrong thinking down.

Reply:
I'm not one to let being wrong get in the way of writing a long comment on github :) hopefully the following is not so wrong that it points people in the wrong direction...@hsorby and I might revise tomorrow.

The basic idea is that the `ImportSource` does two things: the first is it points to a URL (the source) from which a CellML model is expected to be obtainable; the second is that it acts as a container/set/handle for all imported entities which should be obtained from the same *instance* of that source model. This becomes important, for example, when you have sibling components imported from the same instance (variable connections maintained) or different instances (connections severed). As with @hsorby's comment and from #188, I think a bit more of an explanation about what an `ImportSource` is would be useful here. I also now think "encapsulating an external CellML model" is actually incorrect/misleading.

Maybe @hsorby and I can come up with a better docstring tomorrow based on the above and comments on #188...